### ISSUE TITLE
Trashing assets results in infinitely nested trash directories

### ISSUE THREAD

**Issue by lilaconlee** [cypress-io/cypress#2742]
### Current behavior:
Reported from Gitter, trashing assets and trying to download artifacts results in a file too long error:
```
screenshots/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-46b7-83a5-a75b568c4ba9/.Trash-0/files/27869827-fa54-......file name too long
```
### Desired behavior:

Should be able to trash files without recursively creating Trash directories. Alternative could be deleting files when running on CI.

### Steps to reproduce:
Haven't been able to reproduce, but haven't successfully tried on the version of Ubuntu reported. Was unable to reproduce on 18.10.

### Versions

Cypress 3.1.1
Ubuntu 16.04

**Comment by jennifer-shehane** [comment]
Maybe related comment involving recursive Trash dir problem: https://github.com/cypress-io/cypress/issues/2392#issuecomment-416383258

**Comment by jennifer-shehane** [comment]
Also a related issue with permissions accessing Trash dir, not sure you want to tackle this with it though: https://github.com/cypress-io/cypress/issues/1281

**Comment by lilaconlee** [comment]
Opted to delete files on Linux instead of trashing them. Seems like the lowest maintenance solution as trashing in Linux seems inconsistent, but lemme know if that doesn't seem like a reasonable trade-off.

**Comment by jennifer-shehane** [comment]
The code for this is done, but this has yet to be released. We'll update this issue and reference the [changelog](https://on.cypress.io/changelog) when it's released.

**Comment by brian-mann** [comment]
Released in [`3.1.2`](https://docs-staging.cypress.io/guides/references/changelog.html#3-1-2).

**Comment by electrofLy** [comment]
Why is the option to fully delete contents is not available on Windows as well? Our CI server (running on Windows) disk space is getting full as all videos of all projects are sent to the Recycle Bin, which results in ~50GB of videos for 1 week.
